spec/controllers/tags_controller_spec.rb

```diff
@@ -107,6 +107,17 @@
           get :show, :name => 'foo', :format => :mobile
           expect(response).to be_success
         end
+
+        it "returns the post with the correct age" do
+          post2 = eve.post(
+            :status_message,
+            text:       "#what #yes #hellyes #foo tagged second post",
+            public:     true,
+            created_at: @post.created_at - 1.day
+          )
+          get :show, name: "what", max_time: @post.created_at, format: :json
+          expect(JSON.parse(response.body).first["guid"]).to eq(post2.guid)
```

Comment:
I'd maybe add
```ruby
expect(JSON.parse(response.body).length).to be(1)
```
(or `expect(JSON.parse(response.body).size).to be(1)` @jhass ?)